verify-matching-service-test-tool/src/main/resources/legacy/special-name-case.json

```diff
@@ -0,0 +1,43 @@
+  {
+    "hashedPid": "8fbfdd035a92daf890673d9ff8f50def3aad66ea42026c581bd73735250ba706",
+    "levelOfAssurance": "LEVEL_1",
+    "matchId": "default-match-id",
+    "matchingDataset": {
+      "addresses": [
+        {
+          "internationalPostCode": "GB1 2PF",
```

Comment:
Is this a deliberate choice based on a decision about how we'll use `internationalPostCode`? Last time I spoke about it was said we would leave it blank...